packages/core/data-transfer/lib/register.ts

```diff
@@ -179,7 +179,7 @@ const createTransferController =
             const payload = JSON.stringify({
               uuid,
               data: data ?? {},
-              error: e,
+              error: e?.message || 'Unknown error',
```

Comment:
I'm pretty sure we only want to be returning the message rather than the whole stack, but we may want to define some error codes instead of strings to assist in the communications.

Reply:
That's interesting, we can definitely improve the error protocol. Let's include that in the next iteration :slightly_smiling_face: 
I think it could benefit not only the remote transfer but the whole feature's error handling :eyes: